pkg/tenant/api/v1/api.go

```diff
@@ -180,13 +181,19 @@ type PageQuery struct {
 }
 
 type Organization struct {
-	ID       string `json:"id" uri:"id"`
-	Name     string `json:"name"`
-	Owner    string `json:"owner"`
-	Domain   string `json:"domain"`
-	Projects int    `json:"projects"`
-	Created  string `json:"created"`
-	Modified string `json:"modified"`
+	ID        string `json:"id" uri:"id"`
+	Name      string `json:"name"`
+	Owner     string `json:"owner"`
```

Comment:
I think we should get rid of the `Owner` field, I know it was already there but an organization can have multiple members with the `Owner` role, so it doesn't really make sense anymore.

Reply:
The Figma designs display an owner, and that's currently what appears on the organization page so should we leave this put for now? The `getOwner` helper method used in `OrganizationDetails` returns the first member with that role for now.

Reply:
Oh, I completely forgot about that. Can we create a spike story to reassess what the owner field should be?

Reply:
We can leave the field in for right now.